services/server/src/server/services/storageServices/SourcifyDatabaseService.ts

```diff
@@ -535,7 +535,7 @@ export class SourcifyDatabaseService
     return response;
   };
 
-  validateVerificationBeforeStoring(verification: Verification): boolean {
+  validateVerificationBeforeStoring(verification: VerificationExport): boolean {
     if (
       verification.compilation.runtimeBytecode === undefined &&
       verification.compilation.creationBytecode === undefined
```

Comment:
probably this is confusing, in here we should check that we should skip only if runtime and creation match are both `undefined`